Simplified/Book/UI/NYPLBookCellDelegate.h

```diff
@@ -1,5 +1,5 @@
 #if FEATURE_AUDIOBOOKS
-@import NYPLAudiobookToolkit;
```

Comment:
The `NYPLBookCellDelegate.h` file is now in the bridging header, importing the `NYPLAudiobookToolkit` here will result in an error with message `Module not found`, probably caused by the order of compiling. Therefore, I forward-declared the class `DefaultAudiobookManager` here to fix this issue.